internal/gtou/payment.go

```diff
@@ -12,6 +14,9 @@ func (c *Converter) newPayment(pymt *bill.Payment) error {
 	}
 	if pymt.Instructions != nil {
 		ref := pymt.Instructions.Ref.String()
+		if pymt.Instructions.Ext == nil || pymt.Instructions.Ext[untdid.ExtKeyPaymentMeans].String() == "" {
+			return fmt.Errorf("payment must contain payment means extension")
```

Comment:
Same approach here as previous, would be great to hack the validation Errors map to explain which field is missing.